tests/smoke_tests/run_smoke_test.py

```diff
@@ -312,6 +345,28 @@ async def _wait_for_process_to_finish_and_retrieve_logs(
             client_python_path="examples.fedprox_example.client",
             config_path="tests/smoke_tests/fedprox_config.yaml",
             dataset_path="examples/datasets/mnist_data/",
+            seed=42,
+            client_metrics_checkers=[
+                LossChecker(1.0138, LossType.CHECKPOINT, MetricType.TRAINING),
+                LossChecker(1.0138, LossType.BACKWARD, MetricType.TRAINING),
+                LossChecker(0.0000, LossType.PROXIMAL, MetricType.TRAINING),
```

Comment:
Based on the config for the smoke tests it looks like the initial proximal weight is set to 0.0. If we're only running one server round, this means that the loss contribution is sort of "ignored." For the smoke tests, I'd recommend we set it to something like 0.01 so that this loss is non-zero. Let me know if you think that would be a problem.